js/iframe-coordinator/webpack.config.js

```diff
@@ -20,6 +20,17 @@ module.exports = {
           options: { debug: true }
         }
       },
+      {
+        test: /\.worker\.(js|ts)$/,
+        exclude: [/node_modules/],
+        use: {
+          loader: "worker-loader",
```

Comment:
This is a pretty awesome loader.  Allows a web-worker to be included into the baseline via a BLOB and avoids the network call and need to ask library users to deploy a worker along with our library.